src/core/icebergConfig.cmake.in

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
```

Comment:
The name of this file is icebergConfig.cmake.in and in the puffin/ folder we have puffinConfig.cmake.in Wouldn't it make sense to name this to coreConfig.cmake.in to keep consistency?

Reply:
This is the package name which is used by `find_package(iceberg CONFIG REQUIRED)`
and `find_package(puffin CONFIG REQUIRED)`. If we use coreConfig.cmake.in then it is weird to call `find_package(core CONFIG REQUIRED)` from an external project. Additionally, I think icebergCore is a little bit awkward. WDYT?